src/store/index.ts

```diff
@@ -89,23 +98,28 @@ export const store = createStore<State>({
     },
   },
   actions: {
+    async setGame({ commit }) {
+      await client.game('portland').then((res) => {
+        commit('SET_GAME_DATA', res)
+      })
+    },
     async setLastTag({ commit }) {
-      await biketagAPI.getTag().then((res) => {
+      await client.getTag().then((res) => {
         commit('SET_LAST_TAG', res.data)
       })
     },
     async setAllTags({ commit }) {
-      await biketagAPI.getTags().then((res) => {
+      await client.getTags().then((res) => {
         commit('SET_ALL_TAGS', res.data)
       })
     },
     async setAllPlayers({ commit }) {
-      await biketagAPI.getPlayers().then((res) => {
+      await client.getPlayers().then((res) => {
         commit('SET_ALL_PLAYERS', res.data)
       })
     },
     async setTopPlayers({ commit }) {
-      await biketagAPI.getPlayers({ sort: 'top' } as getPlayersPayload).then((res) => {
+      await client.getPlayers({ sort: 'top' } as getPlayersPayload).then((res) => {
```

Comment:
I'm not sure that you need to be using async-await here, I will try to remove it.